packages/components/src/ui/card/component.js

```diff
@@ -12,20 +11,21 @@ import { useMemo } from '@wordpress/element';
 /**
  * Internal dependencies
  */
+import { contextConnect } from '../context';
 import { Elevation } from '../elevation';
 import { View } from '../view';
 import * as styles from './styles';
 import { useCard } from './hook';
 
 /**
- * @param {import('@wp-g2/create-styles').ViewOwnProps<import('./types').CardProps, 'div'>} props
+ * @param {import('../context').ViewOwnProps<import('./types').CardProps, 'div'>} props
  * @param {import('react').Ref<any>} forwardedRef
  */
 function Card( props, forwardedRef ) {
 	const { children, elevation, isRounded = true, ...otherProps } = useCard(
 		props
 	);
-	const elevationBorderRadius = isRounded ? ui.get( 'cardBorderRadius' ) : 0;
+	const elevationBorderRadius = isRounded ? '2px' : 0;
```

Comment:
don't we have a JS variable for the default border radius or something like that.

Reply:
Yup, I'm not sure why I didn't use it here 🤦‍♀️